samples/cpp/benchmark_app/utils.cpp

```diff
@@ -612,6 +614,12 @@ std::vector<benchmark_app::InputsInfo> get_inputs_info(const std::string& shape_
             info_map[name] = info;
         }
 
+        if (batch_size > 1 && !is_there_at_least_one_batch_dim) {
```

Comment:
I'm just think: if model was converted with some batch and user specify -b 1, it also can lead to problems. Maybe raise exception always?

Reply:
From other hand it can affects default validation (model batch size == 1, -b 1)

Reply:
this exception will raise only if we have no any information about batch for any input. In this case we cannot figure out batch size and it will be treated as 1 (for FPS counting). So, if model was converted with another batch we **have to** provide layout information, otherwise FPS calculation will be inaccurate. 
BTW, in normal situation layout information should be provided all the time - either via command line or during model conversion. If it is not provided - it's invalid case, we show warning about it. The only reason we don't throw error is for backward compatibility with validations and for OOB case to avoid user confusion.

Reply:
Concluding that said, I think current algorithm is OK, we should not change it.